stdlib/universe/universe.flux

```diff
@@ -4643,3 +4643,126 @@ toTime = (tables=<-) => tables |> map(fn: (r) => ({r with _value: time(v: r._val
 // tags: date/time
 //
 today = () => date.truncate(t: now(), unit: 1d)
+
+// yesterday finds the starting and stopping point of yesterday truncated to the day.
+//
+// ## Examples
+//
+// ### Returns a recording containing the timestamp of the start and the stopping point of yesterday
+// ```no_run
+// option now = () => 2022-01-02T13:45:28Z
+//
+// yesterday()
+// // Returns {start: 2022-01-01T00:00:00.000000000Z, stop: 2022-01-02T00:00:00.000000000Z}
+// ```
+//
+// ### Query data from yesterday
+// ```no_run
+// from(bucket: "example-bucket")
+//     |> range(start: yesterday())
+// ```
+//
+// ## Metadata
+// introduced: NEXT
+// tags: date/time
+
+yesterday = () => {
+    ret = {start: (date.sub(d:1d, from:today())), stop: today()}
+    return ret
+}
+
+
+// Monday will return the a record containing the start and the stopping point of the last monday
+// ## Parameters
+// - d: time.
+//
+// ## Examples
+//
+// ### Find the Monday of that week 
+//
+// ```no_run
+// import "date"
+//
+// d = date.monday(d: 2022-05-10T10:10:00Z)
+//
+// // Returns {start: 2022-05-08T00:00:00.000000000Z stop:2022-05-09T00:00:00.000000000Z }
+// ```
+//
+// ## Metadata
+// tags: date/time
+// 
+monday = (d) => {
```

Comment:
Don't forget, the ask was to have a function like this for every day of the week.

Seems like the function bodies for all 7 could share some common (private) functions to do their work.